corpus/accounts/views.py

```diff
@@ -37,6 +41,34 @@ def signin(request):
                 return redirect("index")
             else:
                 return redirect(next)
+        else:
```

Comment:
Does the form validation itself fail if there is no matching user?

Reply:
Yep